pvlib/pvsystem.py

```diff
@@ -618,6 +611,32 @@ def pvsyst_celltemp(self, poa_global, temp_air, wind_speed=1.0):
         return temperature.pvsyst_cell(poa_global, temp_air, wind_speed,
                                        **kwargs)
 
+    def faiman_celltemp(self, poa_global, temp_air, wind_speed=1.0):
+        """
+        Use :py:func:`temperature.faiman` to calculate cell temperature.
+
+        Parameters
+        ----------
+        poa_global : numeric
+            Total incident irradiance [W/m^2].
+
+        temp_air : numeric
+            Ambient dry bulb temperature [C].
+
+        wind_speed : numeric, default 1.0
+            Wind speed in m/s measured at the same height for which the wind
+            loss factor was determined.  The default value 1.0 m/s is the wind
+            speed at module height used to determine NOCT. [m/s]
```

Comment:
I see N0CT is used in other docstrings, but I don't see a definition anywhere. Could be a follow up issue.

Reply:
NOCT = Nominal Cell Operating Temperature. Maybe spell out the acronym?

We could add a Note with the definition: NOCT is cell temperature in conditions of irradiance 800 W/m2, ambient air temperature 20C, wind speed 1m/s and cell at Voc, reference to IEC 61215-2 but I'm guessing on the part, I can check when in the office tomorrow.

Reply:
@cwhanse I'm happy to copy/paste a Note snippet from you into this PR, but I'll point out that NOCT is also used in the docstrings of `PVSystem.pvsyst_celltemp`, `temperature.pvsyst_cell`, and `temperature.faiman`.  I wonder if instead of including definitions in docstrings, it might be better to have a central list of definitions somewhere in the sphinx docs?  Or just rely on the PVPMC website or PVCDROM (which as [a page](https://www.pveducation.org/pvcdrom/modules-and-arrays/nominal-operating-cell-temperature) on NOCT already)?

Reply:
Ok. Let’s address it separately if at all.

Reply:
@CameronTStark I re-approve this PR for merge :)